qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -25,6 +27,18 @@
 from qiskit.qobj.utils import MeasLevel
 
 
+class ParametricPulseShapes(Enum):
+    """Map the assembled pulse names to the pulse module commands.
+
+    The enum name is the transport layer name for pulse shapes, the
+    value is its mapping to the OpenPulse Command in Qiskit.
+    """
+    gaussian = commands.Gaussian
+    gaussian_square = commands.GaussianSquare
+    drag = commands.Drag
+    constant = commands.ConstantPulse
+
```

Comment:
Is there some way around this structure? I'm thinking of the following scenario:
External party, named ABC, wants to use Qiskit and provide their own pulse shapes. It would be a lot nicer if, a user of ABC, in his application, could do
```
from ABC.parametric_pulses import ABCPulse
```
where `ABC.parametric_pulses.py` contains the parametric pulses defined by ABC, without having to manually register the parametric pulse names in this `Enum` class. Is something like that possible?

Reply:
I would love it if that was possible! Can you think of a way? I couldn't

Reply:
See #3688.